lib/capybara/node/matchers.rb

```diff
@@ -131,6 +152,45 @@ def assert_no_selector(*args)
 
       ##
       #
+      # Asserts that the current_node matches a given selector
+      #
+      #     node.assert_match_selector('p#foo')
+      #     node.assert_match_selector(:xpath, '//p[@id="foo"]')
+      #     node.assert_match_selector(:foo)
+      #
+      # It also accepts all options that {Capybara::Node::Finders#all} accepts,
+      # such as :text and :visible.
+      #
+      #     node.assert_match_selector('li', :text => 'Horse', :visible => true)
+      #
+      # @param (see Capybara::Node::Finders#all)
+      # @raise [Capybara::ExpectationNotMet]      If the selector does not match
+      #
+      def assert_match_selector(*args)
```

Comment:
Do you think that this method name is descriptive enough to differentiate it from others that check descendants of an element instead of an element itself? What about including `self` into method name? e.g. `assert_self_matches_selector`